src/java.base/share/classes/jdk/internal/classfile/impl/BufWriterImpl.java

```diff
@@ -175,6 +189,14 @@ public void writeIntBytes(int intSize, long intValue) {
         }
     }
 
+    public void skip(int skipSize) {
+        int nextOffset = offset + skipSize;
+        if (nextOffset > elems.length) {
+            grow(nextOffset);
+        }
+        this.offset = nextOffset;
```

Comment:
Let's not duplicate code to get rid of a simple addition
```suggestion
        reserveSpace(skipSize);
        this.offset += skipSize;
```